src/ocean_emulators/utils/loss.py

```diff
@@ -67,3 +71,88 @@ def decomposed_mse_mae(
     mae = F.l1_loss(pred, target, reduction="none")
     combined = (mse + mae) / 2
     return combined.mean(dim=(0, 2, 3))
+
+
+class MseDynamic:
+    """A loss function that scales each channel to contribute equally to the loss.
+
+    This uses a rolling estimate of the loss of each channel to scale each
+    channel's loss, discouraging the model from focusing on only a few channels.
+
+    See: https://openathena.slack.com/archives/C08CYM42DT3/p1752275713570969
+    """
+
+    N_WINDOW = 25
+    """Rolling window size to average over. (~number of steps)"""
```

Comment:
nit: I think it's conventional to go in the other order.

```suggestion
    """Rolling window size to average over. (~number of steps)"""
    N_WINDOW = 25
```

Reply:
I agree it is weird, but the [relevant PEP](https://peps.python.org/pep-0257/#what-is-a-docstring) says:

> String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called “attribute docstrings”.

And according to [various folks on SO](https://stackoverflow.com/questions/8820276/docstring-for-variable), this is what Sphinx, IntelliJ, VS Code, etc, support.